index.html

```diff
@@ -7,10 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type="text/javascript">
```

Comment:
this is a good attempt

Reply:
The script is getting called too early, which is why it cannot find the <div> with class `.navbar_links_toggle`

Reply:
`this looks like code`